mkosi.extra/usr/lib/dracut/modules.d/01setup-ironic-python-agent/20-autologin-serial-getty.conf

```diff
@@ -1,3 +0,0 @@
-[Service]
-ExecStart=
-ExecStart=-/sbin/agetty --skip-login --login-options root --keep-baud 115200,38400,9600 %I $TERM
```

Comment:
Why was this auto-login removed?

Reply:
See: https://github.com/sapcc/fedIPA/pull/21#discussion_r1002957012